en/components/general/wpf_to_angular_guide/one_way_binding.md

```diff
@@ -0,0 +1,115 @@
+---
+title: One-way data binding
+_description: WPF to Angular guide topics help you switch from WPF to Angular.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, One-way, binding
+---
+
+## One-way data binding in Angular
+
+One of the most powerful and widely used feature in WPF is data binding. It makes developers' life much easier, by synchronizing the business logic with views and vice versa, without having to write a single line of extra code. Without the power of it, WPF would be just a better-looking WindowsForms. 
+Luckily, Angular does support data binding. There are two types of data binding, one-way binding, and two-way binding. 
+
+### One-way data binding
+
+Data binding in Angular is super easy, and unlike in WPF we don't have to worry about a data context or a view model. All we have to worry about is an HTML file and a typescript file. With any data binding, the first thing you need are properties to data bind to. So let's add a property called `text` into the component class, and set it a simple value.
+In WPF, we need to set the DataContext and bind the property in XAML:
+```csharp
+public class IgniteUIClass
+{
+  public IgniteUIClass()
+  { this.Text = "IgniteUI for Angular"; }
+  
+  public string Text
+  { get; set; }
+}
+...
+public MainWindow()
+{
+  InitializeComponent();
+  this.DataContext = new IgniteUIClass();
+}
+```
+```xml
+<Label Content="{Binding Path=Text, Mode=OneWay}"></Label>
+```
+In Angular, we are directly binding a DOM property to component's property:
+```typescript
+export class SampleComponent implements OnInit {
+  text = 'IgniteUI for Angular';
+
+  constructor() { }
+  ngOnInit() {}
+}
+```
+```html
+<h2>{{ text }}</h2>
+```
+
+### Interpolation
+
+The above code, simply bind an element in the HTML which is h2, to the value of the `text` property. In this case, we are using the `interpolation` to create one-way binding. We do that by typing double curly braces, the name of the property - in our case `text`, and two closing curly braces. Another way to achieve the same result is, to create h2 tag and bind the `text` property to its innerHTML property, again using the `interpolation` syntax:
+```html
+<h2 innerHTML="{{ text }}"></h2>
+```
+There two important things about `interpolation`. First, everything inside curly braces is rendered as a `string`. 
+And second, everything inside the curly braces is referred to `template expression`. This allows to do more complex things, such as a concatenation. For example, let's concatenate some text with the value of the `text` property:
+```html
+<h2>{{"Welcome to " + text }}</h2>
+```
+Using the template expression also allows to data bind to java script properties and methods. For example, we can data bind to the `text` propetry length which will result - 20:
```

Comment:
1. java script -> javascript
2. propetry -> property
3. For example, we can ~~data~~ bind to the `text` property length which will result in 20:

Reply:
Addressed